mechanicalsoup/stateful_browser.py

```diff
@@ -237,7 +237,7 @@ def find_link(self, *args, **kwargs):
         else:
             return links[0]
 
-    def follow_link(self, link=None, *args, **kwargs):
+    def follow_link(self, link=None, return_url_only=False, *args, **kwargs):
```

Comment:
I think it would be confusing for `follow_link` to have an option that makes it not follow a link and instead do something that seems more like a behavior of `find_link`.